InvenTree/InvenTree/tasks.py

```diff
@@ -290,7 +291,7 @@ def update_exchange_rates():
     Rate.objects.filter(backend="InvenTreeExchange").exclude(currency__in=currency_codes()).delete()
 
 
-def send_email(subject, body, recipients, from_email=None):
+def send_email(subject, body, recipients, from_email=None, html_message=None):
```

Comment:
What's the difference between "body" and "html_message" here?

Reply:
HTML code will be displayed as a plain text if the template is sent as the second argument to `send_mail()` function which is the `message` argument (we call it `body` here). `html_message` keyword argument is used to render the html template inside the email body. [reference](https://docs.djangoproject.com/en/3.2/topics/email/)

Reply:
Thanks!